docs/en/operations/settings/settings.md

```diff
@@ -1397,6 +1397,23 @@ Possible values:
 
 Default value: 16.
 
+## insert_distributed_sync {#insert_distributed_sync}
+
+Enables or disables synchronous data insertion into a [Distributed](../../engines/table-engines/special/distributed.md#distributed) table.
+
+By default, when inserting data into a `Distributed` table, the ClickHouse server sends data to cluster nodes in asynchronous mode. When `insert_distributed_sync=1`, the data is processed synchronously, and the `INSERT` operation succeeds only after all the data is saved on all nodes. 
```

Comment:
`is saved on all nodes`
This is probably incorrect. We should check. Probably it sends to at least to one replica for each shard of distributed table.

Reply:
The way it would do inserts in cluster is defined by internal_replication setting. https://clickhouse.tech/docs/en/engines/table-engines/special/distributed/#distributed
We needto improve description of `internal_replication` as well. But I think in another task/PR